README.md

```diff
@@ -29,6 +29,44 @@ A style guide for Sightly, the HTML templating system from Adobe Experience Mana
     <link rel="stylesheet" type="text/css" href="styles.css"/>
     ```
 
+  - [1.2](#1.2) <a name='1.2'></a> **Avoid inline JavaScript or CSS.**
+  
+  Sightly is an HTML domain-specific template language and therefore makes little sense to be used to generate inline JavaScript or CSS. Within those contexts, only expressions can be used but the context always has to be specified explicitly, and block statements are impossible to use as HTML elements have no meaning there. Therefore, the Use-API should be used instead to construct the JSON data, which can then easily be passed to the client by writing that JSON into a data attribute.
+  
+    ```html
+    <!--/* Bad */-->
+    <section id="top-teaser" data-sly-use.teaser="com.example.TeaserComponent">
+        <h2>${teaser.title}</h2>
+        <script>
+            var topTeaserConfig = {
+                skin: "${teaser.skin @ context='scriptString'}",
+                animationSpeed: ${teaser.animationSpeed @ context='number'}
+            }
+        </script>
+    </section>
+ 
+    <!--/* Good */-->
+    <section id="top-teaser" data-sly-use.teaser="com.example.TeaserComponent">
+        <h2 data-teaser-config="${teaser.jsonConfig}">${teaser.title}</h2>
+    </section>
+    ```
+
+  - [1.3](#1.3) <a name='1.3'></a> **Don't write unecessary expressions.**
```

Comment:
I like this new rule a lot! But shouldn't this be added as 3.3 to the Expression Language?